sklearn/ensemble/tests/test_stacking.py

```diff
@@ -561,3 +566,59 @@ def fit(self, X, y):
     msg = "'MyEstimator' object has no attribute 'n_features_in_'"
     with pytest.raises(AttributeError, match=msg):
         stacker.n_features_in_
+
+
+@pytest.mark.parametrize("stack_method", ["auto", "predict"])
+@pytest.mark.parametrize("passthrough", [False, True])
+def test_stacking_classifier_multilabel(stack_method, passthrough):
+    X_train, X_test, y_train, y_test = train_test_split(
+        X_multilabel_r, y_multilabel_r, stratify=y_multilabel_r, random_state=42
+    )
+
+    estimators = [
+        ("knnc", KNeighborsClassifier()),
```

Comment:
If I recall properly, we have two ways of exposing the output of `predict_proba` from classifiers that support `multilabel-indicator`:

- as a list of `n_targets` composed of `ndarray` of 2 columns, such as `KneighborsClassifier`, `RandomForestClassifier`.
- as an `ndarray` of `n_targets` columns, such as `MLPClassifier`.

So we should indeed test this case. I am not sure that the current is/else is currently working as expected for the second case.

Reply:
Actually, in the case that we have a list of ndarray composed of 2 columns, we should remove each time one of the columns.

Indeed, the information is collinear (we already drop of the probability columns indeed). Thus, we need to go from the list of ndarray to the ndarray of shape (n_samples, n_targets) indeed.

Reply:
I'll work on it thanks for the feedback